app/views/evaluation_forms/_form.html.erb

```diff
@@ -19,57 +8,93 @@
         provide instructions that will be shared with your evaluators. 
       </p>
       <div>
-        <%= form.label :title, style: "display: block" %>
-        <%= form.text_field :title %>
+        <%= form.label :title, class: "usa-label" %>
+        <%= form.text_field :title, class: "usa-input", data: {"action": "evaluation-form#validatePresence focusout->evaluation-form#validatePresence"} %>
+        <for>
       </div>
 
-      <div>
-        <%= form.label :challenge_phase, style: "display: block" %>
-        <%= form.number_field :challenge_phase %>
-      </div>
+      <%= inline_error(evaluation_form, :title) %>
 
-      <div>
-        <%= form.label :challenge_id, style: "display: block" %>
-        <%= form.text_field :challenge_id %>
+      <label class="usa-label" for="challenge-select">Select a challenge</label>
+      <div class="usa-hint">Choose the challenge this form will evaluate.</div>
+      <div class="usa-combo-box">
+        <select class="usa-select" name="challenge-select" id="challenge-select" data-action="evaluation-form#handleChallengeSelect">
+          <% current_user.challenge_manager_challenges.each do |challenge| %>
+            <% if challenge.is_multi_phase %>
+              <% challenge.phases.each.with_index(1) do |phase, index| %>
+                <option value="<%= "#{challenge.id}.#{index}.#{phase.end_date.strftime("%m/%d/%Y")}" %>"><%= challenge.title %> - Phase <%= index %></option>              
+              <% end %>
+            <% else %>
+              <option value="<%= "#{challenge.id}.1.#{challenge.end_date.strftime("%m/%d/%Y")}" %>"><%= challenge.title %></option>
```

Comment:
this option isn't preserved when the form is edited again after saving. you may need to add the [`selected` html attribute](https://www.w3schools.com/tags/att_option_selected.asp) to the correct option (and also set the hidden_fields to the correct values).

Reply:
ah yes, I messed around with this a few days ago but forgot about it, will revisit

Reply:
I remembered why I didn't finish, it's because the selected attribute doesn't actually populate the search bar in a combo box. Might have to hack this using javasript. I asked about it in the USWDS slack